sycl/test-e2e/AddressSanitizer/lit.local.cfg

```diff
@@ -5,3 +5,13 @@ config.substitutions.append(
 config.substitutions.append(
     ("%force_device_asan_rt", "env SYCL_PREFER_UR=1 UR_ENABLE_LAYERS=UR_LAYER_ASAN")
 )
+
+# FIXME: DeviceSanitizer only supports DG2 and PVC
+config.unsupported = True
+
+if 'gpu-intel-dg2' in config.available_features:
+  config.unsupported = False
+elif 'gpu-intel-pvc' in config.available_features:
+  config.unsupported = False
+
+print('config.available_features', config.available_features)
```

Comment:
Is this line intended for debugging?

Reply:
Yes, I want see its output on pre CI.

Reply:
Okay, maybe we can print it more formally, or do you plan to remove this line later?

Reply:
I will remove it later